github-dark.css

```diff
@@ -3465,6 +3469,10 @@
   .review-comment .pending-batched-suggestion-label {
     background: #302808 !important;
   }
+  /* Yellow private repo lock */
+  .page-new-repo .octicon-lock {
+    color: #cb4 !important;
```

Comment:
```suggestion
    color: #651 !important;
```

Reply:
I would also make the selectors for this PR and #825 part of the same rule so we dont end up with duplicate color rules with only one selector for same job.